news/2016-03-28-upcoming-talks-on-inclusive-digital-education.html

```diff
@@ -102,17 +102,17 @@ <h3>Upcoming Talks on Inclusive Digital Education</h3>
                 </ul>
                 <p>May 2016:</p>
                 <ul>
-                	<li>May 12 to 14 - <a href="http://universalinclusion.unito.it/en/">UNIversal Inclusion Rights and Opportunities for Persons with Disabilities in the Academic Conext</a>
+                	<li>May 12 to 14 - <a href="https://www.unito.it/gallerie/universal-inclusion-rights-and-opportunities-persons-disabilities-academic-context">UNIversal Inclusion Rights and Opportunities for Persons with Disabilities in the Academic Conext</a>
```

Comment:
Typographical error: "Context" not "Conext"

Reply:
Also the colon after "Inclusion" as in "UNIversal Inclusion: Rights and Opportunities for Persons with Disabilities in the Academic Context"